google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -286,6 +300,16 @@ def bucket bucket_name, skip_lookup: false, user_project: nil
         #   does not exist. For more information, see [How to Host a Static
         #   Website
         #   ](https://cloud.google.com/storage/docs/website-configuration#step4).
+        # @param [String] user_project If this parameter is set to a project ID
```

Comment:
Should this also contains:
```
@param [Boolean, String] user_project If this parameter is set to
```

Reply:
I removed the documentation for the boolean from `create_bucket` because I couldn't think of a scenario in which it made sense (although it's still supported.) If you create the bucket, passing `user_project: true` to bill all subsequent operations on it to the "current" project (the same one that created it) should have no effect, correct?

Reply:
Regarding the docs in `File`, thank you, good catch! Updating...

Reply:
@frankyn Updated.

Reply:
Good point! Thanks Chris.